estilo/palettes/night-owl.vim.yml

```diff
@@ -37,3 +37,6 @@ gray7: '#777777'
 graya: '#aaaaaa'
 purple2: '#2d2c5d'
 hoki: '#5f7e97'
+
+tags: '#5A455B'
+green3: '#637777'
```

Comment:
This is very similar to the "gray" color in the current palettes https://htmlpreview.github.io/?https://github.com/haishanh/night-owl.vim/blob/master/scripts/palettes.html

Reply:
this one has a greenish hue, which still conforms to the palette. I tried gray but it's a bit flat for my taste.